libs/blocks/faas/utils.js

```diff
@@ -190,40 +199,23 @@ const afterYiiLoadedCallback = () => {
   $.each(faasform, (i, form) => {
     $(form).on('change', () => {
       const elements = $('span.required', form);
-      hideDisplay(elementspdfLink);
+      hideDisplay(elements);
     });
   });
 
   $('.faas-form').on('submit', (event) => {
+    removeExtraColumnDivForNext();
+    const nextFirstInput = document.querySelector('.faas-form.next input:not([type=hidden])');
+    if (nextFirstInput && !nextFirstInput.value) {
+      nextFirstInput.parentElement.classList.add('error');
+      nextFirstInput.parentElement.querySelector('.errorMessage').textContent = 'Business email cannot be blank.';
```

Comment:
What about localization?

Reply:
Good found!, I updated this only for fallback. (There originally put this just for fallback for edge case that can't load error message temporally from FaaS API.)